### DISCUSSION TITLE
My solution on #116 Headphone is only 99.9%, but no diff, can some one help?

### DISCUSSION THREAD

**Discussion by proksza-mano** [cssbattle/forum#355] [Q&A] [answered]
I want to get 100%, but for some reason it's only 99.9%.

I can't see any difference, even went and checked my solution on [CSSBattle Previewer 2.0](https://tc70f3.csb.app/), but that showed me the same no difference from the original.

```
<div></div>
<style>
  :root {
  --t: 20px solid #6E91CA;
  }
  body,div{
    display: flex;
    background: #293D7E;
    margin: 0;
  }
  div {
    width: 130px;
    height: 150px;
    border-top: var(--t);
    border-left: var(--t);
    border-right: var(--t);
    border-radius: 50% 50% 0 0;
    margin: 45 auto;
    justify-content: space-between;
    align-items: flex-end;
    &:before,&:after{
      content:'';
      background:#6E91CA;
      width: 20px;
      height: 40px;
      bottom: -10px;
      outline: var(--t);
      position: relative;
      border-radius: 1% 1% 30px 30px;
    }
  }
</style>
```

**Comment by proksza-mano** [comment, marked as answer]
Sorry I found this is caused because CSSBattle uses older chromium and does not support outlines mentioned in other story: #174

**Reply by SelenIT** [reply]
The CSSBattle backend has been updated since that answer, current version supports rounded outlines and it should not be a problem. Also, if it were the cause, the difference would be visible in the CSSBattle Previewer 2.0, and the score would be less than 99.9%.

The real cause of the problem seem to be these thin lines along the bottom curves which often occur when the background "shines through" the semi-transparent pixels at the anti-aliased edge:
<img width="398" alt="Screenshot 2023-09-03 at 16 44 14" src="https://github.com/cssbattle/forum/assets/282754/6fb7de42-1b74-4643-8410-162840934f10">

It can happen not only with outlines, but also with box-shadows etc. Try to change the solution to avoid edges inside the shapes. E.g. with these styles for pseudo-elements your solution gets 100%:
```css
    &:before,&:after{
      content:'';
      background:#6E91CA;
      width: 60px;
      height: 80px;
      margin: -30px -20px;
      border-radius: 20px 20px 30px 30px;
    }
```
